tests.py

```diff
@@ -48,6 +49,17 @@ def test_clean_input(self):
                 "/", data={'password': 'foo', 'ttl': 'hour'}, method='POST'):
             self.assertEqual((3600, 'foo'), snappass.clean_input())
 
+    def test_password_before_expiration(self):
+        password = 'fidelio'
+        key = snappass.set_password(password, 1)
+        self.assertEqual(password, snappass.get_password(key))
+
+    def test_password_after_expiration(self):
+        password = 'open sesame'
+        key = snappass.set_password(password, 1)
+        time.sleep(1.5)
```

Comment:
One other approach could be to look at the expiration in the redis DB itself. Might not be ideal (test is less functional/black-box) but it would certainly be faster. Let me know if you prefer that.